### ISSUE TITLE
Working Directory Issue.

### ISSUE THREAD

**Issue by neurnn** [wikin/AspNetCore.SpaServices.Extensions.Vue#1]
```
var processInfo = new ProcessStartInfo
{
    FileName = isWindows ? "cmd" : "npm",
    Arguments = $"{(isWindows ? "/c npm " : "")}run serve",
    WorkingDirectory = "ClientApp",
    RedirectStandardError = true,
    RedirectStandardInput = true,
    RedirectStandardOutput = true,
    UseShellExecute = false,
};
```
It should be like:
```
var processInfo = new ProcessStartInfo
{
    FileName = isWindows ? "cmd" : "npm",
    Arguments = $"{(isWindows ? "/c npm " : "")}run serve",
    WorkingDirectory = spa.Options.SourcePath,
    RedirectStandardError = true,
    RedirectStandardInput = true,
    RedirectStandardOutput = true,
    UseShellExecute = false,
};
```

It causes **MSB6003, cmd.exe not found: System.IO.DirectoryNotFoundException: ClientApp**.

**Comment by neurnn** [comment]
Additionally, the build target file, SpaRoot property shouldn't be defined at there. it must be in *.csproj file.

```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <SpaRoot>ClientApp\</SpaRoot>
  </PropertyGroup>
  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>

    <PackageId>AspNetCore.SpaServices.Extensions.Vue</PackageId>
    <PackageVersion>0.0.1</PackageVersion>

    <Authors>Miroslav Peterka</Authors>
  .....................
```

To be :
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>

    <PackageId>AspNetCore.SpaServices.Extensions.Vue</PackageId>
    <PackageVersion>0.0.1</PackageVersion>

    <Authors>Miroslav Peterka</Authors>
```

And the target project should be:
```
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <SpaRoot>UI\</SpaRoot>
  </PropertyGroup>
  .....................
  <ItemGroup>
    <PackageReference Include="AspNetCore.SpaServices.Extensions.Vue" Version="0.0.1" />
    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="5.0.7" />
  </ItemGroup>
  .....................
```

Finally, Readme.md file can notify about property that, `SpaRoot`.

**Comment by neurnn** [comment]
I've requested modification.
https://github.com/wikin/AspNetCore.SpaServices.Extensions.Vue/pull/2#issue-680128138

**Comment by wikin** [comment]
@neurnn tnks for tip. I will use it in next release.

**Comment by wikin** [comment]
released 0.0.2
